ads/google/a4a/utils.js

```diff
@@ -110,7 +110,8 @@ export function googleAdUrl(
   const referrerPromise = viewerForDoc(a4a.getAmpDoc()).getReferrerUrl();
   return getAdCid(a4a).then(clientId => referrerPromise.then(referrer => {
     const adElement = a4a.element;
-    const slotNumber = adElement.getAttribute('data-amp-slot-index');
+    window.ampAdGoogleIfiCounter = window.ampAdGoogleIfiCounter || 1;
```

Comment:
I checked this awhile ago and it works if not quoted, I think because the compiler treats all globals as externs. That said, I would encourage quoting the property anyway, so that nobody else asks this question in future.